style.css

```diff
@@ -96,3 +96,272 @@ a {
+  color: #1c1a19;
+}
+
+.top h4 {
+  font-family: crete Round, sans-serif;
+  font-size: 30px;
+  line-height: 52px;
+  text-align: left;
+  vertical-align: top;
+  color: #1c1a19;
+}
+
+.fram-two {
+  margin-top: 114px;
+  width: 375px;
+  height: 568px;
+  margin-left: 24px;
+}
+
+.featured-image img {
+  width: 327px;
+  height: 237px;
+}
+
+.right-block {
+  background-color: #e5e5e5;
+  width: 350px;
+  height: 320px;
+  padding-bottom: 300px;
+}
+
+.right-block h6 {
+  line-height: 24px;
+  font-family: Crete Round, sans-serif;
+  font-size: 18px;
+  padding-top: 10px;
+  color: #091e42;
+  text-align: center;
+  vertical-align: top;
+}
+
+.right-block p {
+  font-size: 12px;
+  font-family: inter, sans-serif;
+  line-height: 16px;
+  color: #091e42;
+  padding-left: 0;
+  margin-right: 40px;
+  padding-right: 24px;
+}
+
+.tags {
+  width: 297px;
+  height: 40px;
+  gap: 12px;
+}
+
+.tags img {
+  width: 49px;
+  height: 40px;
+  gap: 30px;
+  flex-direction: row;
+  padding-top: 20px;
+  padding-bottom: 18px;
+}
+
+.tags button {
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 3%;
+  color: #fff;
+}
+
+.frame43 {
+  width: 375px;
+  height: 2436px;
+  background-color: #fff;
+}
+
+.mask-group-one {
+  width: 327px;
+  height: 110px;
+  background-image: url("image/imgPlaceholderTwo.png");
+  background-repeat: no-repeat;
+  margin-left: 24px;
+  padding-bottom: 386px;
+  background-position-x: right;
+  background-position-y: 120px;
+}
+
+.mask-group-one h1 {
+  padding-top: 200px;
+  padding-left: 16px;
+  padding-right: 16px;
+  width: 295px;
+  height: 81px;
+  color: #fff;
+  font-family: Crete Round, sans-serif;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: left;
+  vertical-align: top;
+  margin-bottom: 5px;
+}
+
+.mask-group-one p {
+  margin-left: 24px;
+  margin-right: 24px;
+  padding-left: 16px;
+  padding-right: 16px;
+  margin-bottom: 0;
+  width: 295px;
+  height: 81px;
+  color: #fff;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  line-height: 124%;
+  text-align: left;
+  vertical-align: top;
+}
+
+.mask-group-one button {
+  width: 85px;
+  height: 40px;
+  background-color: #a9a9a9;
+  color: #fff;
+  gap: 30px;
+  flex-direction: row;
+  font-family: Inter, sans-serif;
+  font-size: 18px;
+  line-height: 20px;
+  line-height: 110%;
+  text-align: left;
+  vertical-align: top;
+}
+
+.bottom {
+  width: 100%;
+  height: 80px;
+  margin-top: 15px;
+  background-color: #ff6b00;
+}
+
+.bottom p {
+  color: #fff;
+  padding-top: 20px;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  line-height: 117%;
+  text-align: center;
+  letter-spacing: 3%;
+  margin: 12px;
+}
+
+.rectangle {
+  width: 375px;
+  height: 1400px;
+  margin-top: 120px;
+  background-color: #1c1a19;
+  margin-left: 24px;
+  margin-bottom: 0;
+}
```

Comment:
- [ ] Kindle remove the margin you're giving the `rectangle` class here. I see it is the reason that section stretches to the right.

See your design
![image](https://user-images.githubusercontent.com/50941074/160246415-2d45c294-8dc0-4265-8971-d2a005410fe2.png)

** The Figma design **
![image](https://user-images.githubusercontent.com/50941074/160246452-d211b729-e6e6-4d8c-99a3-8f42ec53d86d.png)

Reply:
resolved 👍